composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "microsoft/kiota-bundle",
+    "description": "Default implementations for Kiota client setup",
+    "type": "library",
+	"license": "MIT",
+    "authors": [
+        {
+            "name": "Microsoft Graph Client Tooling",
+            "email": "graphtooling@service.microsoft.com"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Microsoft\\Kiota\\Bundle\\": "src/"
+        }
+    },
+	"autoload-dev": {
+		"psr-4": {
+			"Microsoft\\Kiota\\Bundle\\Test\\": "tests/"
+		}
+	},
+    "require": {
+		"php": "^7.4 || ^8.0",
+		"microsoft/kiota-abstractions": "^1.4.0",
+		"microsoft/kiota-authentication-phpleague": "^1.3.1",
```

Comment:
bundles do not contain the authentication package in other languages

Reply:
thanks for catching this! Resolved in https://github.com/microsoft/kiota-bundle-php/pull/1/commits/f01b668a7bbc9dc9c7bffeaa6abc40466a8219c7